main.py

```diff
@@ -193,7 +197,11 @@ async def hello(interaction: discord.Interaction):
 @tree.command(description="See today's problem.")
 @app_commands.checks.cooldown(1, COOLDOWN_SECONDS)
 async def current_challenge(interaction: discord.Interaction):
-    check_submission_channel()
+    if store.__getitem__("submission_channel_id") == 0:
```

Comment:
Why repeat this and not put it in to a function? If you make a change somewhere you will have to remember to make a change everywhere else.